firmware/hw_layer/can_hw.cpp

```diff
@@ -271,12 +281,7 @@ static msg_t canThread(void *arg) {
 			writeStateToCan();
 
 		if (engineConfiguration->canReadEnabled)
-			canRead(); // todo: since this is a blocking operation, do we need a separate thread for 'write'?
-
-		if (engineConfiguration->canSleepPeriodMs < 10) {
```

Comment:
this check didn't make sense - it might only make sense for writes.  future work is to split read/write out on to their own threads so that they aren't coupled

Reply:
self-comment is a sign of code comment opportunity missed in my opinion

Reply:
not in cases of when something was removed?

Reply:
doing this future work first - closing for now